malware/get_samples.py

```diff
@@ -66,7 +66,7 @@ def run_query(url):
         # the response is a 'data' object, which is a list of objects
         resp = json.loads(urlopen(url).read())['data'][0]
     except Exception as e:
-        print('Failed to load "%s": %s' % (md5, str(e)))
+        print('Failed to load "%s": %s' % ('md5', str(e)))
```

Comment:
I guess we could just include "md5" as part of the string instead of adding it in the tuple? Nice catch!